release-notes/5.0/5.0.0-preview3.md

```diff
@@ -0,0 +1,91 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 5.0.0-preview3 released 15 June 2022
+
+This update brings the below changes over the previous release:
+
+### Added
+- Added support for `TDS 8`. To use TDS 8, users should specify Encrypt=Strict in the connection string. Strict mode disables TrustServerCertificate (always treated as False in Strict mode). HostNameInCertificate has been added to help some Strict mode scenarios. [#1608](https://github.com/dotnet/SqlClient/pull/1608)
+
+- Added support to skip the driver-generated Server Prinicipal Name when using SPN and Failover Partner SPN options in a connection string.
```

Comment:
The missing link to the PR.
You can create a section for additional information and link it here.
```suggestion
- Added support server SPN option. [#1607](https://github.com/dotnet/SqlClient/pull/1607)
```